Cargo.toml

```diff
@@ -2,7 +2,7 @@
 name = "notify-server"
 version = "0.1.0"
 edition = "2021"
-authors = ["Maciej Rak <raku@walletconnect.com>"]
+authors = ["Chris Smith <chris@walletconnect.com>"]
```

Comment:
Maybe you can also add a `codeowners` to `.github` with your GitHub tag so that you get notified of every PR

Reply:
Added in 1c56ea2